LogMonitor/src/LogMonitor/ProcessMonitor.cpp

```diff
@@ -239,16 +239,15 @@ size_t formatProcessLog(char* chBuf)
 {
     const char* prefix;
     const char* suffix;
-    if (Utility::CompareWStrings(settings.LogFormat, L"JSON"))
+    if (Utility::CompareWStrings(settings.LogFormat, L"XML"))
     {
-        // {"Source":"Process","LogEntry":{"Logline":"<chBuf>"},"SchemaVersion":"1.0.0"}
-        prefix = "{\"Source\":\"Process\",\"LogEntry\":{\"Logline\":\"";
-        suffix = "\"},\"SchemaVersion\":\"1.0.0\"}\n";
-    } else {
-        // <Log><Source>Process</Source><LogEntry><Logline><chBuf>Z</Logline></LogEntry></Log>
         prefix = "<Log><Source>Process</Source><LogEntry><Logline>";
         suffix = "</Logline></LogEntry></Log>\n";
-    }    
+    }
+    else {
+        prefix = "{\"Source\":\"Process\",\"LogEntry\":{\"Logline\":\"";

```

Comment:
nit: trying to understand what has changed other than flipping the if/else blocks? Can include some description on the PR...

Reply:
ok, I get it now. With this change, does it mean when the format is `custom` it will now add JSON string? should we handle the _custom_ scenario explicitly?

Reply:
@profnandaa I have updated the description.

Reply:
@profnandaa This will need a discussion to re-format the config file from the current format to something like this:

```
{
  "LogConfig": {
    "sources": [
      {
        "type": "ETW",
        "eventFormatMultiLine": false,
        "providers": [
          {
            "providerName": "Microsoft-Windows-WLAN-Drive",
            "providerGuid": "DAA6A96B-F3E7-4D4D-A0D6-31A350E6A445",
            "level": "Information"
          }
        ]
      },
      {
        "type": "File",
        "directory": "c:\\inetpub\\logs",
        "filter": "*.log",
        "includeSubdirectories": true
      }
    ],
	"logFormat": "custom",
	"customLogFormat": [
		"ETW": "{'TimeStamp':'%TimeStamp%', 'source':'%Source%', 'Severity':'%Severity%', 'ProviderId':'%ProviderId%', 'ProviderName':'%ProviderName%', 'EventId':'%EventId%', 'EventData':'%EventData%'}",
		"File": "{'TimeStamp':'%TimeStamp%','message':%Message%,'source':%Source%,'fileName':%FileName%}",
		"EventLog": "{'TimeStamp':'%TimeStamp%', 'source':'%Source%', 'Severity':'%Severity%','message':%Message%}"
		"Process": "{'TimeStamp':'%TimeStamp%', 'source':'%Source%', 'Severity':'%Severity%','message':%Message%}"
	]
  }
}
```

cc @iankingori

Reply:
Got it, `custom` is sort of a sub-type of `json`. Thanks!